API.md

```diff
@@ -808,6 +810,24 @@ const boolean = Joi.boolean();
 boolean.validate(true, (err, value) => { });
 ```
 
+#### `boolean.truthy(value)`
+
+Allows for additional strings to be considered valid boolean values in addition to default 'truthy' strings: ['true', 'yes', '1', 'on']. Accepts a string or an array of strings.
+
+```js
+const boolean = Joi.boolean().truthy('Y');
+boolean.validate('Y', (err, value) => { });
```

Comment:
Doing `boolean.validate('Y', (err, value) => { }); // valid` or something else might help to show that this works.